doc/files/package.json.md

```diff
@@ -98,9 +98,8 @@ If a url is provided, it will be used by the `npm bugs` command.
 You should specify a license for your package so that people know how they are
 permitted to use it, and any restrictions you're placing on it.
 
-The simplest way, assuming you're using a common license such as BSD-3-Clause
-or MIT, is to just specify the standard SPDX ID of the license you're using,
-like this:
+If you're using a common license such as BSD-2-Clause or MIT, add a
+current SPDX license identifier for the license you're using, like this:
```

Comment:
Can you make this a link to whichever piece of documentation of SPDX you think is most approachable for developers who are unfamiliar with it? I had never heard of it before you got in touch with me, and I have an enterprise background, so I imagine many other people will be confused as well.